.github/workflows/release.yaml

```diff
@@ -0,0 +1,93 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+  workflow_dispatch:
+    inputs:
+      imageTag:
+        description: Base Image Tag (before the :)
+        type: string
+        required: false
+        default: rancher/rancher-cleanup
+
+jobs:
+  shellcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Run shellcheck
+        run: |
+          sudo apt install shellcheck
```

Comment:
I am +1 with @ericpromislow. Considering we follow a pullrequest workflow, I think it would be better to move all tests to a dedicated workflow executed on branch change and pullrequest.

Running test during a release process doesn't make sense to me because issues should have been catched before during pullrequest review.
1. It just slow down the release process
2. It introduce a risk of failing the release due to temporary